index.html

```diff
@@ -0,0 +1,109 @@
+    <title>Creative Engineers</title>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/speakers.css">
+    <link rel="stylesheet" href="css/footerHome.css">
+</head>
+<body>
+    <header>
+        <ul class="container">
+            <li><a href="https://www.facebook.com/mohamed.abdelmohsen.9655/"><i class="fa-brands fa-facebook-f"></i></a></li>
+            <li><a href="https://twitter.com/mohamed20163858"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="index.html">English</a></li>
+            <li><a href="index.html">My Page</a></li>
+            <li><a href="index.html">Logout</a></li>
+        </ul>
+    </header>
+    <nav class="container">
+        <a href="index.html"><img src="images/logo_transparent_2.png" alt="logo icon"></a>
+        <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="index.html">Program</a></li>
+            <li><a href="index.html">Join</a></li>
+            <li><a href="index.html">Sponsor</a></li>
+            <li><a href="index.html">News</a></li>
+        </ul>
+        <p><a href="index.html">CE Campaign</a></p>
+    </nav>
+    <main class="home" id="home-page">
+        <section class="landing-page container">
+            <img src="images/Union.png" alt="menu-icon">
+            <p>"Hello!Future Engineers"</p>
+            <h1>CREATIVE ENGINEERS GLOBAL JOB FAIR 2022</h1>
+            <p>We are a group of talented engineers in different engineering fields but sharing a common goal is to help people start their own busniess by help them build their projects system using the nowadays technologies in a very creative , realiable and fashion way come and join us today to gain your desirable experience you dreamed of!</p>
+            <p>2022.10.15(MON) ~ 17(WEN)</p>
+            <p>	@ m1 stage, faculty of engineering, Alexandria university, Egypt</p>
+        </section>
+        <section class="main-program">
+            <div class="white title">
+                <h2>Main Program</h2>
+                <hr>
+            </div>
+            <div class="cards">
+                <div class="card">
+                    <img src="images/lecture.png" alt="lecture icon">
+                    <p>Lecture</p>
+                    <p>Listen to the speakers from our company that work in different fields and aspects</p>
+                </div>
```

Comment:
- [Optional] Hi @mohamed20163858, I think this is an article. For semantic reasons, I'll prefer us to use an article element(`<artcle></article>`) instead of the generic div element(`<div></div>`). Remember we have five different articles hence you'll require 5 article elements on this page.